tests/9-soc/murax/CMakeLists.txt

```diff
@@ -39,5 +39,13 @@ add_vivado_target(
     CLOCK_PINS io_mainClk
     CLOCK_PERIODS 10.0
     )
+
+add_vivado_pnr_target(
+    NAME murax_basys_vivado_pnr
+    PARENT_NAME murax_basys
+    CLOCK_PINS io_mainClk
```

Comment:
Can we just use an SDC / XDC file instead of putting this in the build rule?

Reply:
That is an option, but most of the existing symbiflow tests just use a PCF right now.

Reply:
I logged https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1000